From 4ce3e8a857f2540909e1c23b73beef62eb3ab5fa Mon Sep 17 00:00:00 2001
From: jordi fita i mas <jfita@infoblitz.com>
Date: Sat, 11 Apr 2020 22:37:20 +0200
Subject: [PATCH] Add Qt5PrintSupport and KF5CoreAddons dependencies to CMake
 config

Reviewers: staniek

Reviewed By: staniek

Subscribers: staniek

Tags: #kreport

Differential Revision: https://phabricator.kde.org/D28736
---
 src/KReportConfig.cmake.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/KReportConfig.cmake.in b/src/KReportConfig.cmake.in
index 7d74122b..9828fcc1 100644
--- a/src/KReportConfig.cmake.in
+++ b/src/KReportConfig.cmake.in
@@ -20,6 +20,8 @@ include(CMakeFindDependencyMacro)
 find_dependency(Qt5Core @REQUIRED_QT_VERSION@)
 find_dependency(Qt5Widgets @REQUIRED_QT_VERSION@)
 find_dependency(Qt5Xml @REQUIRED_QT_VERSION@)
+find_dependency(Qt5PrintSupport @REQUIRED_QT_VERSION@)
+find_dependency(KF5CoreAddons @REQUIRED_KF5_VERSION@)
 find_dependency(KPropertyWidgets @REQUIRED_KPROPERTY_VERSION@)
 
 include("${CMAKE_CURRENT_LIST_DIR}/KReportTargets.cmake")
-- 
GitLab

